stan/math/prim/prob/std_normal_cdf.hpp

```diff
@@ -29,17 +29,14 @@ template <typename T_y>
 inline return_type_t<T_y> std_normal_cdf(const T_y& y) {
   using T_partials_return = partials_return_t<T_y>;
   using std::exp;
-
   static const char* function = "std_normal_cdf";
-
-  T_partials_return cdf(1.0);
+  check_not_nan(function, "Random variable", y);
 
   if (size_zero(y)) {
-    return cdf;
+    return 1.0;
```

Comment:
This is much better than defining a variable and returning it.  Plus, the partials return isn't even the right type---it'd just get promoted to the return type anyway. 

[optional]
This can just be `1`---the compiler doesn't need the `.0` unless there's a chance of ambiguity.  It will promote appropriately at compile time.